aptos-move/framework/src/natives/account.rs

```diff
@@ -16,29 +16,36 @@ use move_deps::{
 use smallvec::smallvec;
 use std::collections::VecDeque;
 
-pub fn native_create_signer(
-    context: &mut NativeContext,
+pub fn native_create_address(
+    _context: &mut NativeContext,
     ty_args: Vec<Type>,
     mut arguments: VecDeque<Value>,
 ) -> PartialVMResult<NativeResult> {
     debug_assert!(ty_args.is_empty());
     debug_assert!(arguments.len() == 1);
 
-    let address = pop_arg!(arguments, AccountAddress);
-    let cost = native_gas(context.cost_table(), NativeCostIndex::CREATE_SIGNER, 0);
-    Ok(NativeResult::ok(cost, smallvec![Value::signer(address)]))
+    let cost = GasCost::new(super::cost::APTOS_CREATE_ADDRESS, 1).total();
```

Comment:
Can you leave a TODO here to move this into the gas schedule once we move it from upstream move into our repo?